tests/spread/integration/iputils-ping/task.yaml

```diff
@@ -0,0 +1,6 @@
+summary: Integration tests for iputils-ping
+
+execute: |
+  rootfs="$(install-slices iputils-ping_bins)"
+
+  chroot "${rootfs}/" ping -V
```

Comment:
Following the recommendations from https://github.com/canonical/chisel-releases/pull/447#pullrequestreview-2533580554, could we instead test with at least something like `ping -c 2 localhost`?